### DISCUSSION TITLE
Version 1.76 on the latest version of Visual Studio 17.5.1 causes problems when opening Python projects

### DISCUSSION THREAD

**Discussion by proffix4** [community/community#48899] [Copilot Conversations]
### Select Topic Area

Bug

### Body

Version 1.76 in the latest version of Visual Studio 17.5.1 causes problems when opening Python projects. There is an infinite "Solution Preparation". After removing the extension, everything becomes normal. The previous version of Visual Studio (before today's update) was also OK. Please fix this problem

**Comment by marcveilleux** [comment]
I get the exact same problem with any VB.Net solutions: There is an infinite "Solution Preparation".
Only C# solutions are working fine with GitHub Copilot v 1.76 and VS Pro 2022 17.5.1
